server/src/commands.ts

```diff
@@ -280,6 +280,12 @@ program
   .option("-q, --queued", "Run job asynchronously", false)
   .action(createJobAction("metabase:jobs:collection"))
 
+program
+  .command("creer-role-management-metabase")
```

Comment:
```suggestion
  .command("metabase:role-management:create")
```

Harmonisons les commandes entre jobs.ts et commands.ts